index.js

```diff
@@ -0,0 +1,147 @@
+const day = document.getElementById("js-day"),
+ date = document.getElementById("js-date"),
+ month = document.querySelector(".js-month");
+
+ today = new Date(),
+ MONTH = today.getMonth(),
+ YEAR = today.getFullYear(),
+ PAINTING = "painting";
+
+// 현재 요일, 일자
+function currentDay() {
+    const week = new Array('SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT');
+    day.innerText = week[today.getDay()];
+    date.innerText = today.getDate()
+}
+
+//현재 월, 년
+function currentMonth() {
+    const months = new Array("JAN", "FEB", "MAR", "APR", "MAY", "JUN",
+    "JUL", "AUG", "SEP", "OCT", "NOV", "DEC");
+    const thisMonth = months[today.getMonth()];
+    const thisYear = today.getFullYear();
+    month.innerText = `${thisMonth} ${thisYear}`
+}
+
+// 달력 행&열 추가, 일자 채우기
+function makeCalendar() {
+    let lastDate = new Date(YEAR, MONTH+1, 0).getDate(); // 다음달 0번째 일
+    let firstDay = new Date(YEAR, MONTH, 1).getDay(); //firstDay부터 일자 출력하기 위함
+    let calendar = document.querySelector(".js-calendarBody");
+
+    let row = calendar.insertRow();
+    for(i = 0; i < firstDay; i++){
+        cell = row.insertCell();
+    } // 공백 만들기
+    for(i=1; i <= lastDate; i++){
+        if(firstDay != 7){
```

Comment:
```suggestion
        if (firstDay !== 7) {
```

비교 연산을 할 때 `!==` or `===`을 사용하는 것을 권장합니다.
Strict equality와 Equality를 함께 알아보시면 좋을 듯 합니다.

- [MDN: Strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)
- [MDN: Equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)
- [javascript-double-equals-vs-triple-equals](https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a)